### PR TITLE
Stop rspamd if no anti-virus and no anti-spams are enabled

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5563

### CODE REVIEW

File: filter/etc/e-smith/events/actions/nethserver-mail-filter-disable_rspamd [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+use esmith::ConfigDB;
+my $cdb = esmith::ConfigDB->open() || die("Could not open ConfigDB");
+
+my $spamStatus = $cdb->get_prop('rspamd', 'SpamCheckStatus') || 'enabled';
+my $virusStatus = $cdb->get_prop('rspamd', 'VirusCheckStatus') || 'enabled';
+
+if (($spamStatus eq 'disabled') && ($virusStatus eq 'disabled'))
+    {
+        $cdb->set_prop('rspamd', 'status', 'disabled');
```
Comment (DavidePrincipi) [inline filter/etc/e-smith/events/actions/nethserver-mail-filter-disable_rspamd]: Changing the DB values in an action is not recommended. The admin or the Server Manager has to set the prop, then run signal-event.

**Comment by stephdl** [comment]
yes you are right this is another feature request by @markVnl and after shared with him, it concerns a lot more services like clamd and  probably redis

**Comment by stephdl** [comment]
moved this PR to another issue https://github.com/NethServer/dev/issues/5551

**Comment by stephdl** [comment]
obsoleted by https://github.com/NethServer/nethserver-mail/pull/70

**Pull request closed without merging**